apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -235,7 +234,7 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
 
   const getWorkspaceCodeHelper = (
     retryCount: number,
-    hiddenWorkspace: Workspace | undefined
+    hiddenWorkspace: ExtendedWorkspace | undefined
```

Comment:
Why did this become `ExtendedWorkspace` and not `GoogleBlockly.Workspace`?

Reply:
Good question, and I'm not sure! Just changed it to the latter.